nixos/modules/programs/steam.nix

```diff
@@ -6,6 +6,14 @@ let
   cfg = config.programs.steam;
 
   steam = pkgs.steam.override {
+    extraPackages = pkgs: [
```

Comment:
should this be `extraPkgs` instead?

Reply:
(judging from https://github.com/NixOS/nixpkgs/blob/440405af3b5f52d35362c2b3501888d7c728e2b4/pkgs/games/steam/fhsenv.nix#L3)

Reply:
Whoops, that's absolutely correct, fixed.